statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -748,117 +1072,180 @@ def forecast_interval(self, y, steps, alpha=0.05):
 # VARResults class
 
 
-class VARResults(VARProcess):
-    """Estimate VAR(p) process with fixed number of lags
+class VARResults(VARProcess, tsbase.TimeSeriesModelResults):
+    """The results instances of a fitted VAR(p) model.
 
     Parameters
     ----------
-    endog : array
-    endog_lagged : array
+    model : VAR model instance
+        A fitted VAR model
+    Y : array
+        2d array that contains the endogenous variables
+    X : array
+        2d array that contains the lagged endogenous variables.
     params : array
-    sigma_u : array
+        The VAR(p) coefficients. The shape is ((k_trend + k_ar*neqs) x neqs)
+        Each equation is expected to be in a column. The order of the
+        parameters are trend variables first, then first lag of the first
+        equation variable, the second lag of the second equation variables,
+        etc.
+    cov_resid : array
+        The covariance matrix of the residuals. This is the equation by
+        equation covariance.
     lag_order : int
-    model : VAR model instance
-    trend : str {'nc', 'c', 'ct'}
-    names : array-like
-        List of names of the endogenous variables in order of appearance in `endog`.
-    dates
-
+        The number of lags used.
 
     Returns
     -------
     **Attributes**
-    aic
-    bic
-    bse
-    coefs : ndarray (p x K x K)
-        Estimated A_i matrices, A_i = coefs[i-1]
-    cov_params
-    dates
-    detomega
+    coefs : ndarray (k_ar x neqs x neqs)
```

Comment:
rename to `ar_coefs` similar to ARMA, 
we ended up there with coefs instead of params, IIRC because params is lag polynomial parameterization.